Pod/Classes/Categories/UIApplication+PreferredContentSizeCategoryMock.h

```diff
@@ -0,0 +1,15 @@
+//
+//  UIApplication+PreferredContentSizeCategoryMock.h
+//  AGSnapshotHelper
+//
+//  Created by Adam Grzegorowski on 15/08/2017.
+//  Copyright © 2017 Grupa Allegro. All rights reserved.
+//
+
+@import UIKit;
+
+@interface UIApplication (PreferredContentSizeCategoryMock)
+
+@property (nonatomic, strong, nullable) UIContentSizeCategory mockedContentSizeCategory;
```

Comment:
Do you want to add documentation later?

Reply:
Documentation is always nice. Same for tests 😉

Reply:
@aleksandergrzyb  Added